### PR TITLE
Switch session url to port

### PR DESCRIPTION
**Pull request by sipsma**
Signed-off-by: Erik Sipsma <erik@sipsma.dev>

Related to: https://github.com/dagger/dagger/issues/3830 (see `DAGGER_SESSION_PORT or DAGGER_SESSION_URL?` section of issue description)

Slight tweak which results in our SDKs only ever being expected to connect over 127.0.0.1

### CODE REVIEW

**Review by marcosnils — APPROVED** [review]
LGTM!

**Comment by helderco** [comment]
Heads up, I'm going to update the Python part.

**Review by helderco — APPROVED** [review]
:shipit:

**Comment by sipsma** [comment]
Will wait for a review on the NodeJS code from @slumbering or @TomChv before merge

**Review by slumbering — APPROVED** [review]
LGTM in the nodejs side.

**Pull request merged**
